auth-web/src/components/auth/CreateUserProfileLanding.vue

```diff
@@ -12,7 +12,7 @@
           <v-col lg="8" class="pt-0 pb-0">
             <div class="view-header user-profile-header">
               <h1>Director Search Account Access</h1>
-              <p class="mb-0">You've been invited to administer the '@@replace@@' Director Search Account at the BC Registry. Create your user profile and log in to access this account</p>
+              <p class="mb-0">You've been invited to administer the '----replace----' Director Search Account at the BC Registry. Create your user profile and log in to access this account</p>
```

Comment:
Not sure I understand what's happening with the --replace--... is this actually a binding, or a placeholder?

Reply:
changed that, thats a placeholder for the account name, now its working